docs/docs/workflows/update-manifest.md

```diff
@@ -52,17 +52,123 @@ Don't define other values except the `SecurityVersion` value for a package, as M
+To deploy a new manifest, your user must have a [role assigned that contains the `UpdateManifest` action](define-manifest.md#roles). If multiple users have such a role assigned, each of them must [acknowledge](#acknowledging-an-update) the new manifest.
 
 ## Deploying an update
 
 Use the CLI to deploy an update, specifying the client certificate and private key of a user with appropriate permissions:
 
 ```bash
-marblerun manifest update update-manifest.json $MARBLERUN --cert=admin-cert.pem --key=admin-key.pem --era-config=era.json
+marblerun manifest update apply update-manifest.json $MARBLERUN --cert=user-cert.pem --key=user-key.pem --era-config=era.json
 ```
 
-On success, no message will be returned and your MarbleRun logs should highlight that an update manifest has been set. On error, the API endpoint will return an error message. If you receive `unauthorized user`, MarbleRun either received no client certificate over the TLS connection, or you used the wrong certificate.
+On success, no message will be returned and your MarbleRun logs should highlight whether the update manifest has been set or further acknowledgements are required. On error, the API endpoint will return an error message. If you receive `unauthorized user`, MarbleRun either received no client certificate over the TLS connection, or you used the wrong certificate.
+
+## Acknowledging an update
+
+Use the CLI to acknowledge a full manifest update, specifying the client certificate and private key of a user with appropriate permissions:
+
+```bash
+marblerun manifest update update-manifest.json $MARBLERUN --cert=user-cert.pem --key=user-key.pem --acknowledge --era-config=era.json
+```
+
+### Example: Multi-party update
+
+The following gives an example of a full manifest update with multiple parties:
+
+Assume the following `Users` and `Roles` were defined in the Manifest:
+
+```javascript
+    "Users": {
+        "alice": {
+            "Certificate": "-----BEGIN CERTIFICATE-----\nMIIFZTCCA02gAwIBAgIUANHwS8RM0PUDl9htA+yWJx9WqucwDQYJKoZIhvcNAQEL\nBQAwQjELMAkGA1UEBhMC=\n-----END CERTIFICATE-----\n",
+            "Roles": [
+                "UpdateManifest"
+            ]
+        },
+        "bob": {
+            "Certificate": "-----BEGIN CERTIFICATE-----\nMIIFZTCCA02gAwIBAgIUJvtF7KRsunTmWVtpU9198HUxyLEwDQYJKoZIhvcNAQEL\nBQAwQjELMAkGA1UEBhMC=\n-----END CERTIFICATE-----\n",
+            "Roles": [
+                "UpdateManifest"
+            ]
+        }
+    },
+    "Roles": {
+        "UpdateManifest": {
+            "ResourceType": "Manifest",
+            "Actions": ["UpdateManifest"]
+        }
+    },
+```
+
+1. Alice applies an update by uploading an update manifest:
+
+    ```bash
+    $ marblerun manifest update apply new_manifest.json $MARBLERUN ---cert alice.crt --key alice-private.pem
+    Coordinator verified 
+    Loading client certificate
+    Creating client
```

Comment:
```suggestion
```
Debug statements that slipped through in a PR, will be removed by https://github.com/edgelesssys/marblerun/pull/356